File: web/app/model/pam/MonthlyWorkView.js
Ext.define('iFlat.model.pam.MonthlyWorkView', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'monthlyWorkView.pbName', mapping: 'pbName', type: 'string'},
        {name: 'monthlyWorkView.month', mapping: 'month', type: 'date'},
        {name: 'monthlyWorkView.status', mapping: 'status', type: 'string'},
    ]
});

File: src/main/java/com/iflat/pam/entity/MonthlyWorkView.java
package com.iflat.pam.entity;

import java.util.Date;

/**
 * Created by tyriv on 2016/8/8.
 */
public class MonthlyWorkView {

    private String pbName;
    private Date month;
    private String status;

    public String getPbName() {
        return pbName;
    }

    public void setPbName(String pbName) {
        this.pbName = pbName;
    }

    public Date getMonth() {
        return month;
    }

    public void setMonth(Date month) {
        this.month = month;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}
